woodwork/table_accessor.py

```diff
@@ -921,6 +922,9 @@ def mutual_information_dict(
                 to measure accurately and will return a NaN value. Must be
                 non-negative. Defaults to 25.
             random_seed (int): Seed for the random number generator. Defaults to 0.
+            max_nunique (int): The maximum number of unique values for large categorical columns (> 800 unique values).
```

Comment:
Nit: Could you change this to "The *total* maximum number of unique values for *all* large categorical columns"